src/main/java/seedu/address/model/person/Person.java

```diff
@@ -31,23 +32,25 @@ public Person(Name name, Phone phone, Email email, Address address) {
         requireAllNonNull(name, phone, email, address);
         this.name = name;
         this.phone = phone;
+        this.nokPhone = new NokPhone();
```

Comment:
This constructor is for add command.
Since add command must include adding of the nok phone, i think there is no need for a default nok phone "000" because the tutor would be required to specify a nokphone number. +
maybe we can add NokPhone nokPhone to the constructor parameter instead?